Atomic repo clean up

andresgalante opened the pull request:
Hi!

This PR is a clean up:

- It empties the templates for PRs and issues
- Creates a new READ ME
- Adds license file
- Adds license headers to all our scss
- Moves core CSS to `patternfly.scss`

I've also already removed the branches we don't use.

@srambach @jgiardino @bleathem can you please review it? if you want we can have a small meeting and I sow you the changes.

@LHinson Can you please read the first part of the READ ME? it's here https://github.com/andresgalante/patternfly-atomic/blob/clean-up/README.md

Thanks a lot guys! 😄

jgiardino commented on README.md:
```diff
@@ -1,64 +1,192 @@
+
+Think of PatternFly as a "skinned" version of [Bootstrap v4](http://v4-alpha.getbootstrap.com/) with additional components and customizations.
+
+This repository is javascript free, you'll find only find CSS here.
```
Minor typo here: "you'll ~~find~~ only find CSS here."

srambach replied:
And the comma needs to be a semicolon :)

jgiardino commented on README.md:
```diff
@@ -1,64 +1,192 @@
+
+### Coding style guidelines
+
+Please read the coding style guidelines for PatternFly 4.
```
Were you going to add a link to the guidelines here?

jgiardino commented on README.md:
```diff
@@ -1,64 +1,192 @@
+#### Watch for changes and re-generate Pattern Lab
 
-## Packaged Components
+To watch for changes, re-generate the front-end, and server it via a BrowserSync server,  type:
```
Just a minor typo: "and serve ~~r~~ it via a..."

jgiardino commented on README.md:
```diff
@@ -1,64 +1,192 @@
+
+___
+
+Built with :heart: by the [PattrenFly Community](https://www.patternfly.org/community/) and supported by the Red Hat UXD Team.
```
Doh: "PattrenFly"

jgiardino commented on README.md:
```diff
@@ -1,64 +1,192 @@
+___
+
+Built with :heart: by the [PattrenFly Community](https://www.patternfly.org/community/) and supported by the Red Hat UXD Team.
+
```
What do you think about adding the command for building the github page here?

jgiardino commented on source/scss/_pf-shame.scss:
```diff
@@ -1,4 +1,19 @@
+//
+// Patternfly Shame
+// --------------------------------------------------
+// The nest fot all our shamefull code.
```
Just more types: "The nest ~~fot~~ **for** all our ~~shamefull~~ **shameful** code."

jgiardino commented:
Looks great, Andres! Thanks for doing this!

srambach commented on README.md:
```diff
@@ -1,64 +1,192 @@
+
+Please read the coding style guidelines for PatternFly 4.
+
+The guidelines are a work in progress and a live document, if you have any suggestion please [contact the PatternFly community](#communication-channels) or send a PR.
```
Another semicolon - or maybe even better, a period - instead of comma here. Also, "suggestions" should be plural.

srambach commented:
Great improvements, @andresgalante !

andresgalante commented:
@srambach @jgiardino I've fixed all the typos and added the github pages instructions. For now it's ok, but I feel we will need to flesh the instructions out to make them actually usable.

Thanks a lot for reviewing it 😄

andresgalante commented:
@jgiardino @srambach @bleathem I've solved the conflicts, can you please merge this one?
